models/api.models.js

```diff
@@ -1,7 +1,24 @@
 const db = require("../db/connection");
+const articles = require("../db/data/test-data/articles");
 
 exports.fetchTopics = () => {
   return db.query("SELECT * FROM topics;").then(({ rows: topics }) => {
     return topics;
   });
 };
+
+exports.fetchArticleById = (article_id) =>{
+  return db
+  .query(`
+    SELECT * FROM articles
```

Comment:
well done for preventing against sql injection here and good handling of the error process!